boards/arc/nsim/support/nsim_sem.props

```diff
@@ -30,7 +30,7 @@
 	nsim_isa_smart_stack_entries=8
 	mpu_sid_option=1
 	mpu_regions=16
-	mpu_version=4
```

Comment:
We don't want to modify existing nsim_sem board as we till going to support MPUv4. Moreover, by bumping MPU version without syncing everything else with latest SEM template(s) we introduce the configuration which may exist only in simulation but not in real HW.

As we don't introduce any new code to handle MPUv8 separately I guess it's fine to keep it untested for now.